consumer/dragonboard/README.md

```diff
@@ -1,23 +1 @@
-| <img src="https://i.imgur.com/4a5GXRd.png" data-canonical-src="https://i.imgur.com/4a5GXRd.png" width="200" height="130" /><br> **DragonBoard 410c** | Board based on Qualcomm® Snapdragon™ 410E processor  | [Documentation](dragonboard410c/)<br>|
-
-</div>
+---
title: DragonBoard Documentation
permalink: /documentation/consumer/dragonboard/
---
Welcome to the official documentation for Consumer Edition 96Boards, these documents are written by the [96Boards](https://www.96boards.org) team at [Linaro](http://www.linaro.org) with community contributions and links to third-party content.

***

## Contents - Start Here

Select your Consumer Edition DragonBoard 96Boards device to access all product specific resources. You may also use the links below to compare, and explore a list of 96Boards Consumer Edition extras, this includes instructions for unique board configurations and fun projects.

- DragonBoard Generic Guides

<div style="overflow-x:scroll;" markdown="1">

| 96Boards                                         | About                                                  | Options                    |
|:------------------------------------------------:|:------------------------------------------------------:|:--------------------------:|
| <img src="https://www.96boards.org/product/ce/rb5-platform/images/rb5-front-sd.png?raw=true" data-canonical-src="https://www.96boards.org/product/ce/rb5-platform/images/rb5-front-sd.png?raw=true" width="200" height="130" /><br> **Qualcomm Robotics RB5** | Board based on Qualcomm® QRB5165 SoC  | [Documentation](dragonboard/qualcomm-robotics-rb5/)<br> |
| <img src="https://www.96boards.org/product/ce/rb3-platform/images/db845c-front-sd.png?raw=true" data-canonical-src="https://www.96boards.org/product/ce/rb3-platform/images/db845c-front-sd.png?raw=true" width="200" height="130" /><br> **Qualcomm RB3** | Board based on Qualcomm® Snapdragon™ 845 processor  | [Documentation](dragonboard/dragonboard845c/)<br> |
| <img src="https://github.com/96boards/documentation/blob/master/consumer/dragonboard/dragonboard820c/additional-docs/images/images-board/sd/dragonboard820c-front-sd.png?raw=true" data-canonical-src="https://github.com/96boards/documentation/blob/master/consumer/dragonboard/dragonboard820c/additional-docs/images/images-board/sd/dragonboard820c-front-sd.png?raw=true" width="220" height="200" /><br> **DragonBoard 820c** | Board based on Qualcomm® APQ8016E processor  | [Documentation](dragonboard820c/)<br> |
| <img src="https://i.imgur.com/4a5GXRd.png" data-canonical-src="https://i.imgur.com/4a5GXRd.png" width="200" height="130" /><br> **DragonBoard 410c** | Board based on Qualcomm® Snapdragon™ 410E processor  | [Documentation](dragonboard410c/)<br>|

</div>



```

Comment:
formatting needs to be fixed, weird `^M` characters appearing in-place of  regular `<enter>/<return>`

Reply:
Hi Sahaj,

I had originally used Windows Desktop to commit the changes. I have a feeling the characters got added automatically somehow. If I open the files in the browser, I do not see the characters.
If I re-commit each file individually, seems like the characters go away. I have done this for all the files.